tests/ha/test_control_pod_restart.py

```diff
@@ -383,3 +401,286 @@ def test_failover_control_pod_kubectl(self):
+                'output': wr_output}
+        thread_wr = threading.Thread(target=self.ha_obj.event_s3_operation, args=(event,),
+                                     kwargs=args)
+        thread_wr.daemon = True  # Daemonize thread
+        thread_wr.start()
+        LOGGER.info("Step 4: Successfully started WRITEs with variable sizes objects in background")
+        time.sleep(HA_CFG["common_params"]["20sec_delay"])    # delay to allow s3bench installation
+
+        LOGGER.info("Step 5: Perform READs and verify DI on the written data in background")
+        output_rd = Queue()
+        args = {'s3userinfo': list(users.values())[1], 'log_prefix': test_prefix_read,
+                'nclients': 5, 'nsamples': 50, 'skipwrite': True, 'skipcleanup': True,
+                'setup_s3bench': False, 'output': output_rd}
+        thread_rd = threading.Thread(target=self.ha_obj.event_s3_operation, args=(event,),
+                                     kwargs=args)
+        thread_rd.daemon = True  # Daemonize thread
+        thread_rd.start()
+        LOGGER.info("Step 5: Successfully started READs and verified DI on the written data in "
+                    "background")
+
+        LOGGER.info("Control pod %s is hosted on %s node", self.control_pod_name, self.control_node)
+
+        failover_node = self.system_random.choice([ele for ele in self.host_worker_list if ele !=
+                                                   self.control_node])
+        LOGGER.debug("Fail over node is: %s", failover_node)
+
+        event.set()
+        LOGGER.info("Step 6: Failover control pod %s to node %s and check cluster status",
+                    self.control_pod_name, failover_node)
+        pod_yaml = {self.control_pod_name: self.modified_yaml}
+        resp = self.ha_obj.failover_pod(pod_obj=self.node_master_list[0], pod_yaml=pod_yaml,
+                                        failover_node=failover_node)
+        assert_utils.assert_true(resp[0], resp)
+        LOGGER.info("Step 6: Successfully failed over control pod to %s. Cluster is in good state",
+                    failover_node)
+
+        event.clear()
+        self.restore_pod = True
+        LOGGER.info("Step 7: Verify if IAM users %s are persistent across control pod failover",
+                    uids)
+        for user in uids:
+            resp = self.rest_iam_user.get_iam_user(user)
+            assert_utils.assert_equal(resp.status_code, const.Rest.SUCCESS_STATUS,
+                                      f"Couldn't find user {user} after control pod failover")
+            LOGGER.info("User %s is persistent: %s", user, resp)
+        LOGGER.info("Step 7: Verified all IAM users %s are persistent across control pod "
+                    "failover", uids)
+
+        LOGGER.info("Waiting for background IOs thread to join")
+        thread_wr.join()
+        thread_rd.join()
+        thread_del.join()
+        LOGGER.info("Step 8.1: Verify status for In-flight DELETEs during control pod "
```

Comment:
LOGGER.info("Step 8: Verify status for In-flight READs/WRITEs/DELETEs during control pod failover to %s", failover_node)